kratos/sources/kratos_parameters.cpp

```diff
@@ -209,12 +210,70 @@ Parameters::Parameters()
 /***********************************************************************************/
 /***********************************************************************************/
 
+nlohmann::json Parameters::ReadFile(std::string file_name) {
```

Comment:
Input is always CammelCase
```suggestion
nlohmann::json Parameters::ReadFile(std::string FileName) {
```